packages/components/src/search-control/stories/index.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import SearchControl from '..';
+
+const meta: ComponentMeta< typeof SearchControl > = {
+	title: 'Components/SearchControl',
+	component: SearchControl,
+	argTypes: {
+		onChange: { action: 'onChange' },
+		onClose: { action: 'onClose' },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof SearchControl > = ( {
+	onChange,
+	onClose,
+	...props
+} ) => {
+	const [ value, setValue ] = useState< string >();
+
+	return (
+		<SearchControl
+			{ ...props }
+			value={ value }
+			onClose={ onClose }
```

Comment:
Can the `onClose` prop be passed together with `...props` ?

Reply:
🙈